server/controllers/assess/assessmentsController.ts

```diff
@@ -116,4 +117,24 @@ export default class AssessmentsController {
       })
     }
   }
+
+  riskInformation(): RequestHandler {
```

Comment:
I think I missed this for the prison information, but _technically_ I think I'd like to see this in it's own controller, with a `show` action. It is more of a style thing though really, and given we're not doing any CRUD actions, is probably not worth all the extra boilerplate.

Reply:
You did say a similar thing for prison information. 
I think it's a decent solution but I was at a loss for a name for it... maybe 'supporting information' controller or something?

Reply:
Yeah, it could be `supportingInformation` which then accepts the name of the type of supporting information, so either `prison-information` or `risk-information`, with a conditional that fetches and renders the relevant information. Then the URL would be:

`/asssessments/:id/supporting-information/:category`

Whaddya reckon?